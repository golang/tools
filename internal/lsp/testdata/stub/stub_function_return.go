package stub

import (
	"io"
)

func newCloser() io.Closer {
	return closer{} //@suggestedfix("c", "refactor.rewrite")
}

type closer struct{}
