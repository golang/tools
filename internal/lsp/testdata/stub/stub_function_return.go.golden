-- suggestedfix_stub_function_return_8_9 --
package stub

import (
	"io"
)

func newCloser() io.Closer {
	return closer{} //@suggestedfix("c", "refactor.rewrite")
}

type closer struct{}

// Close implements io.Closer
func (closer) Close() error {
	panic("unimplemented")
}

